view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -109,8 +108,16 @@ define(
                     groupTypes: self.getAvailableCardTypeAltCodes(),
 
                     onChange: function (state) {
-                        // isValid is not present on start
-                        if (typeof state.isValid !== 'undefined' && state.isValid === false) {
+                        if (!!state.isValid) {
```

Comment:
why !! ?

Reply:
With this check we can do the same as we did previously with the `typeof state.isValid !== 'undefined' && state.isValid === false`

Reply:
TBH I learned it from the checkout guys, tested it and really does the same